app/db.py

```diff
@@ -71,10 +76,15 @@ def drop_ohlcv_table():
 
 def need_recent_data(symbol):
     """ if we have data from today in the db we don't need more recent data"""
-    today = datetime.today()
-    day = date(today.year, today.month, today.day)
-    delta_days =  day - get_start_date(symbol)
-    if delta_days != 0: 
-        return True 
-    else:
+    todays_date = datetime.today()
+    todays_day = date(todays_date.year, todays_date.month, todays_date.day)
```

Comment:
`date.today()` is available in addition to `datetime.today()`.  see https://docs.python.org/2/library/datetime.html#datetime.date.today